### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pythonpete32**
I've added a few notes on setting up a Linux env as well as updated some of the links. I didn't mess with the text too much so it is still a little outdated

### CODE REVIEW

**Comment by stellarmagnet** [comment]
This looks good to me. @ottodevs what do you say?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,22 +32,91 @@ contribute:
...
-- <https://gist.github.com/izqui/2a02c337aafa225c403de9c988cb78a0>
+- <https://wiki.aragon.org/>
+- <https://hack.aragon.org/>
+- <https://hack.aragon.org/docs/aragonos-intro.html>
+- <https://hack.aragon.org/docs/tutorial>
 - <https://github.com/aragon/aragon.js/blob/master/docs/APP.md>
 - <https://github.com/aragon/aragon.js/blob/master/docs/WRAPPER.md>
 
-The WIP gist is probably the most helpful but also the most incomplete. If you follow the instructions, you'll run into some issues with some assumptions made about the APM permissions. As far as design, the app.md and wrapper.md are the most helpful for the front end, and the hack.md gives the best overview. The aragonOS.md details the solidity code and how the core of their system works.
+The [Aragon App tutorial](https://hack.aragon.org/docs/tutorial) is probably the most helpful. As far as design, the app.md and wrapper.md are the most helpful for the front end and the hack gives the best overview. The aragonOS.md details the solidity code and how the core of their system works.
```
Comment (chadoh) [inline CONTRIBUTING.md]: We don't need to do this now, but I would like to see this commentary incorporated into the bulleted list itself.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+
+`nvm install 10.15.3`
+
+**Warning:** `node` version 12 is not compatible with Aragon Cli, `10.15.3` works well but the latest LTS version should work.
+
+#### 3. Install Aragon CLI
+
+`npm i -g @aragon/cli`
+
+Since version 6.0.0 ipfs is not including with the cli so install it with
```
Comment (e18r) [inline CONTRIBUTING.md]: ~including~ included

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+
+`aragon ipfs install`
+
+#### 4. configure cli
+Finally, if you are not using frame as you signing provider (frame is highly recommended) you need to set up a private key for use with the cli.
+
+Create a new key and get some test net Eth from the rinkeby faucet
```
Comment (e18r) [inline CONTRIBUTING.md]: ~test net~ testnet
~Eth~ ETH
~rinkeby~ Rinkeby

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+
+Some times the ~/.aragon file is not included in the installation which is a problem because that's where your private key goes.
+
+`cd ~/.aragon`
+
+If you get file not found run the dev chain first aragon devchain and try again
```
Comment (e18r) [inline CONTRIBUTING.md]: please rephrase this

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+Since version 6.0.0 ipfs is not including with the cli so install it with
+
+`aragon ipfs install`
+
+#### 4. configure cli
+Finally, if you are not using frame as you signing provider (frame is highly recommended) you need to set up a private key for use with the cli.
+
+Create a new key and get some test net Eth from the rinkeby faucet
+
+Some times the ~/.aragon file is not included in the installation which is a problem because that's where your private key goes.
```
Comment (e18r) [inline CONTRIBUTING.md]: ~Some times~ Sometimes
~file~ folder
comma before which

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+```
+{
+ "rpc": "https://rinkeby.aragon.network",
+ "keys": [
+ "put-your-priv-key-here"
+ ]
+}
+```
+
+now test out your configuration by launching a DAO on rinkeby
```
Comment (e18r) [inline CONTRIBUTING.md]: ~rinkeby~ Rinkeby

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+
+#### 3. Install Aragon CLI
+
+`npm i -g @aragon/cli`
+
+Since version 6.0.0 ipfs is not including with the cli so install it with
+
+`aragon ipfs install`
+
+#### 4. configure cli
```
Comment (e18r) [inline CONTRIBUTING.md]: ~cli~ the CLI

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -32,22 +32,91 @@ contribute:
...
+Finally, if you are not using frame as you signing provider (frame is highly recommended) you need to set up a private key for use with the cli.
```
Comment (e18r) [inline CONTRIBUTING.md]: ~cli~ CLI

**Comment by stellarmagnet** [comment]
@e18r, so we can get this merged, can you please go ahead and make the language changes yourself?

**Pull request merged**